parlai/core/metrics.py

```diff
@@ -145,6 +178,11 @@ def __init__(self, opt):
         if nltkbleu is not None:
             # only compute bleu if we can
             self.metrics_list.append('bleu')
+        if rouge is not None:
+            # only compute rougr if we can
+            self.metrics_list.append('rouge-1')
+            self.metrics_list.append('rouge-2')
+            self.metrics_list.append('rouge-l')
```

Comment:
Do you mind making the L capital everywhere? The 1 and l looks so close it would be nice for it to be more obvious

Reply:
That is a great idea!